services/venice-router/src/main/java/com/linkedin/venice/router/stats/RouterHttpRequestStats.java

```diff
@@ -82,37 +80,29 @@ public class RouterHttpRequestStats extends AbstractVeniceHttpStats {
   private final Sensor tardyRequestRatioSensor;
   private final Sensor throttleSensor;
   private final Sensor badRequestSensor;
-  private final LongCounter requestSensorOtel;
 
   /** latency metrics */
   private final Sensor latencySensor;
   private final Sensor healthyRequestLatencySensor;
   private final Sensor unhealthyRequestLatencySensor;
   private final Sensor tardyRequestLatencySensor;
   private final Sensor throttledRequestLatencySensor;
-  private final DoubleHistogram latencySensorOtel;
 
   /** retry metrics */
   private final Sensor errorRetryCountSensor;
-  private final LongCounter retryTriggeredSensorOtel;
   private final Sensor allowedRetryRequestSensor;
-  private final LongCounter allowedRetryRequestSensorOtel;
   private final Sensor disallowedRetryRequestSensor;
-  private final LongCounter disallowedRetryRequestSensorOtel;
   private final Sensor retryDelaySensor;
-  private final DoubleHistogram retryDelaySensorOtel;
```

Comment:
I see that we're getting rid of a portion of the boilerplate in the form of the class fields, but we still have a lot of boilerplate in the rest of the class in terms of metric registration and usage.

I think we should do a brainstorm session to see if there is a reasonable path within reach to improve the boilerplate.